File: CUSTOR.EICOnline.Angular/src/app/model/sector.ts
export class SectorModel {
  public SectorId: number;
  public Description: string;
  public DescriptionAlias: string;
  public DescriptionEnglish: string;
  public DescriptionEnglishAlias: string;
  public EconomicSector: number;

  constructor(SectorId?: number, Description?: string, DescriptionAlias?: string, DescriptionEnglish?: string, DescriptionEnglishAlias?: string, EconomicSector?: number) {
    this.SectorId = SectorId;
    this.Description = Description;
    this.DescriptionAlias = DescriptionAlias;
    this.DescriptionEnglish = DescriptionEnglish;
    this.DescriptionEnglishAlias = DescriptionEnglishAlias;
    this.EconomicSector = EconomicSector;
  }
}

File: CUSTOR.EICOnline.DAL/EntityLayer/Sector.cs
using System.Collections.Generic;
using CUSTOR.EICOnline.DAL.EntityLayer;

namespace CUSTOR.EICOnline.DAL
{
  public partial class Sector
  {
    public Sector()
    {
      SubSector = new HashSet<SubSector>();
    }

    public int SectorId { get; set; }
    public string Description { get; set; }
    public string DescriptionAlias { get; set; }
    public string DescriptionEnglish { get; set; }
    public string DescriptionEnglishAlias { get; set; }
    public int EconomicSector { get; set; }

    public ICollection<SubSector> SubSector { get; set; }
    public Project Project { get; set; }
  }

  public partial class SectorLookup
  {
    public SectorLookup()
    {
    }

    public int SectorId { get; set; }
    public string Description { get; set; }
    public string DescriptionAlias { get; set; }
    public string DescriptionEnglish { get; set; }
    public string DescriptionEnglishAlias { get; set; }
    public int EconomicSector { get; set; }
  }
}